manifests/d/DWANGO/NAIR/latest/DWANGO.NAIR.locale.ja-jp.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DWANGO.NAIR
+PackageVersion: latest

```

```suggestion
PackageVersion: 1.1.20240216-1
```
Please update file name and path to match this change.